gateway/upstreamer/push/notifier.go

```diff
@@ -48,10 +49,16 @@ func (w *Notifier) MakeStartHook(ctx context.Context, frequency time.Duration) f
 			PushEndpoint: server.PushEndpoint(),
 		}
 
-		if sp.Load, err = p.CPUPercent(); err != nil {
+		pct, err := p.CPUPercent()
+		if err != nil {
 			return err
 		}
 
+		// Use the maxproc to get a percentage between 0 and 100
+		cores := float64(runtime.GOMAXPROCS(0))
```

Comment:
Store gomaxproc in a var once for all. It's not free to check that

Reply:
It's done once before the ping loop is started.